db/l2db/l2db.go

```diff
@@ -267,20 +267,22 @@ func (l2db *L2DB) addTxs(txs []common.PoolL2Tx, checkPoolIsFull bool) error {
 	}
 	// Query begins with the insert statement
 	query := queryInsertPart
+	const onConflictQuery = ` ON CONFLICT ON CONSTRAINT tx_id_unique DO UPDATE SET
```

Comment:
Could you explain better how do you update the transaction item in tx-pool and what are the 'updateful' fields?

Reply:
Fields, that can be updated are toAccountIndex, toHezEthereumAddress, toBjj, maxNumBatch, signature, type. Other fields are for atomic txs or take part in transaction id generation. Also, tx state cannot be updated from PUT request